lib/actions/request_review.js

```diff
@@ -0,0 +1,42 @@
+const { Action } = require('./action')
+const _ = require('lodash')
+
+const createRequestReview = async (context, number, reviewers) => {
+  return context.github.pullRequests.createReviewRequest(
```

Comment:
Should there be a try catch here so that we can catch the error for when the reviewer is not a collaborator? I feel we should handle it if the request will not complete properly upstream in flex when it this call throws an error. When that happens, we can end up with additional error logs which becomes noise since it is a "known" error.

Reply:
I've instead chosen to check if the users are collaborator first before requesting review